### PR TITLE
Used the 'result' function defined in this script

### PR DESCRIPTION
**Pull request by moodmosaic**

### CODE REVIEW

File: Furl.fsx [hunk 1]
```diff
@@ -38,4 +38,5 @@ let post url headers body =
     |> result
 
 let bodyText (resp : HttpResponseMessage) =
-    resp.Content.ReadAsStringAsync().Result
+    resp.Content.ReadAsStringAsync ()
+    |> result
```
Comment (moodmosaic) [inline Furl.fsx:42]: FWIW, this should work too

```f#
let bodyText (resp : HttpResponseMessage) =
    ()
    |> resp.Content.ReadAsStringAsync
    |> result
```

**Comment by ploeh** [comment]
Thank you for your interest in Furl, but... how does this change improve the code?

**Comment by moodmosaic** [comment]
Perhaps, this change makes the code in this script more consistent (?) After all, isn't `result` used in a similar way in all the other places in the script?

To put it differently, I was wondering why `result` wasn't used there... And that's why I've sent this boring PR...

**Comment by moodmosaic** [comment]
I think we should close this one, since I'm now pretty sure (after reading your comment in #2) that there's a reason why `result` wasn't used in this particular place.

**Comment by ploeh** [comment]
The trivial reason is that the current code is shorter by a few characters...

**Pull request closed without merging**